lib/pbench/server/api/resources/upload_api.py

```diff
@@ -0,0 +1,192 @@
+import os
+import humanize
+import tempfile
+import hashlib
+from pathlib import Path
+from flask_restful import Resource, abort
+from flask import request, jsonify
+from werkzeug.utils import secure_filename
+
+ALLOWED_EXTENSIONS = {"xz"}
+
+
+class HostInfo(Resource):
+    def __init__(self, server_config, logger, api):
+        self.server_config = server_config
+        self.api = api
+        self.logger = logger
+
+    def get(self):
+        try:
+            response = jsonify(
+                dict(
+                    message=f"{self.server_config.get('user')}@{self.server_config.get('host')}"
+                    f":{self.server_config.get('pbench-receive-dir-prefix')}-002"
+                )
+            )
+        except Exception:
+            self.logger.exception(
+                "There was something wrong constructing the host info"
+            )
+            abort(500, message="There was something wrong with your request")
```

Comment:
Feel free to create an issue since this is not really your code changes ...

Perhaps just report back to the client, "`INTERNAL ERROR`"?